bokeh/util/browser.py

```diff
@@ -19,8 +19,10 @@
 #-----------------------------------------------------------------------------
 
 # Standard library imports
-from os.path import abspath
 import webbrowser
+from os.path import abspath
+from typing import Type, Union, cast
+from typing_extensions import Literal, Protocol
```

Comment:
@enricapq, you will have to add `typing_extensions` to `REQUIRES` in `setup.py`.